### ISSUE TITLE
Invalid two property sets with the same name

### ISSUE THREAD

**Issue by Moult** [Autodesk/revit-ifc#624]
A consultant found this issue in their model that prevented us from extracting data reliably (since property set names are meant to be unique). https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcUniqueDefinitionNames.htm

```
#53341= IFCRELDEFINESBYPROPERTIES('1Uu6iqdKn1Z9w6zLx9ztMR',#42,$,$,(#53321),#53331);
#53345= IFCRELDEFINESBYPROPERTIES('2vyRDgSbn6sg5xOo8dP$_j',#42,$,$,(#53321),#53339);
#53331= IFCPROPERTYSET('1F85KEDPr8IAemQWyJwMg0',#42,'Data','',(#53326,#53327,#53328,#53329,#53330));
#53339= IFCPROPERTYSET('1qt96ZHpL08AWSTYAaMd4B',#42,'Data','',(#656,#657,#658,#53333,#53334,#53335,#53336,#53337,#53338));
```

They are using Revit 2020, latest applicable release on Github. It is a large project and I cannot ask them to test on Revit 2023.

**Comment by AngelVelezSosa** [comment]
Do they have a user defined property set called Data assigned to instance and type of an element that exports to Ifc as an instance entity only (no type entity?)

**Comment by Moult** [comment]
Yes, it has a user defined property set called Data assigned to the instance as well as the type. There is a type entity.
